src/pages/components/search/usage.mdx

```diff
@@ -61,46 +75,221 @@ to aid the user in finding content.
+| Medium (md) | 40 / 2.5        | This is the default size and the most commonly used size. When in doubt, use the medium size.                                                       |
+| Large (lg)  | 48 / 3          | Use when there is a lot of space to work with. The large size is typically at a global level when the user is searching content within a page view. |
+
+<Row>
+<Column colLg={8}>
+
+![Fixed search sizes](images/search-sizes-fixed.png)
+
+</Column>
+</Row>
+
+#### Fluid input height
+
+There is only one input height at 64px and it is visually larger than the fixed
+heights.
+
+<Row>
+<Column colLg={8}>
+
+![Fluid search size](images/search-sizes-fluid.png)
+
+</Column>
+</Row>
+
+Placement
+
+Place the search component within a user interface where users expect to find it
+and where it is appropriate for how it is being used in context, whether at a
+global level, page-specific level, or within a component.
+
+The fixed search can be arranged on the grid to align the container with other
+components containers. Search can also hang to align with other hanging
+component containers on the same page, depending on the use case.
+
+<DoDontRow>
+  <DoDont caption="Do align the fixed search container verticall with other text on the page.">
+
+![Do align the fixed search container vertically with other text on the page.](images/search-do-fixed.png)
+
+</DoDontRow>
+
+<DoDontRow>
+  <DoDont caption="Do hang the fluid search container with other hanging components.">
+
+![Do hang the fluid search container with other hanging components.](images/search-do-fluid.png)
+
+</DoDontRow>
+
+## Content
+
+### Main elements
+
+#### Placeholder text
+
+Useful and short text hinting at what the user can search for. For example,
+“Search for networks or devices.”
+
+#### Field text
+
+- Key words entered by the user to find a search result.
+- Keep the field text concise and relate the field text as closely to the result
+  you want to find.
+
+### Further guidance
+
+For further content guidance, see Carbon’s
+[content guidelines](https://carbondesignsystem.com/guidelines/content/overview/).
+
+## Behaviors
+
+### States
+
+The search component has four states: enabled, focus, filled, and disabled.
+
+<Row>
+<Column colLg={12}>
+
+<Tabs>
+
+<Tab label="Fixed">
+
+![Search states](images/search-states-fixed.png)
+
+</Tab>
+
+<Tab label="Fluid">
+
+![Search states](images/search-states-fluid.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+### Interactions
+
+#### Mouse
+
+Click on the search field input to start typing. Once a user starts typing, a
+Close icon (‘x’) will appear and provides a way to clear the input field by
+clicking on the Close icon.
+
+#### Keyboard
+
+Press `Enter` to submit text as a search term, or press `Esc` to clear the
+search field. Once a user starts typing, a Close icon (‘x’) will appear and
```

Comment:
Close would not be capitalized here since its not a proper noun? Unless its in a code snippet as the title of the icon.
```suggestion
search field. Once a user starts typing, a Close icon (‘x’) will appear and
```

Reply:
Correct, this should not be capitalized. 👍